mlflow/utils/databricks_utils.py

```diff
@@ -183,8 +183,22 @@ def is_in_databricks_repo_notebook():
         return False
 
 
+DATABRICKS_VERSION_FILE_PATH = "/databricks/DBR_VERSION"
+
+
+def get_databricks_runtime_version_string():
```

Comment:
```suggestion
def get_databricks_runtime_version():
```

`_string` seems redundant.